### ISSUE TITLE
Grand Theft Auto V v1.0.505.2 (Update 10) Cracked Incl. 12 DLCs

### ISSUE THREAD

**Issue by dm104** [Guad/GTACoop#28]
I m using the above version and when i press f9 and search for servers there is no server list inside is this because nobody is hosting a server for the newer cracked version on kat.cr?

**Comment by dm104** [comment]
Do i need to be loged in into social club because thise version of the game doesnt log you in or let you register but multiplayer files are there.

**Comment by Guad** [comment]
Check your firewall

**Comment by dm104** [comment]
It doesnt block anything and i updatet to 573 and still no servers inside.

**Comment by Guad** [comment]
Does the script crash or it's just an empty menu?

**Comment by dm104** [comment]
Its just an empty menu but does this work on a pirated version (cracked or is this meant for original only?)

**Comment by white-energy** [comment]
I will try this and report back.

**Comment by dm104** [comment]
Also updated to:
https://kat.cr/grand-theft-auto-v-update-1-573-1-t11819364.html
https://kat.cr/grand-theft-auto-v-update-1-573-1-fix-t11821883.html
Stil doesnt work.

**Comment by white-energy** [comment]
Works fine.

**Comment by dm104** [comment]
Its the script blocking my ip from my country? did you download the version that i use?

**Comment by white-energy** [comment]
Yes, i've downloaded the exact version you're using and everything's fine so it must be something wrong with your script files or firewall..

**Comment by dm104** [comment]
hmmm i installed it on an completley new drive still the same -_- u tihnk that his has something to do with the system?

**Comment by dm104** [comment]
And do i need 100% save games because i only completet few missions till now.

**Comment by white-energy** [comment]
This probably has something to do with Network

**Comment by dm104** [comment]
So i completley uninstaled my windows 10 64 Bit in UEFI mode and all drives in GPT format and i will now see if this works or not.

**Comment by dm104** [comment]
Tryed it on a completley new instalation and still no servers when i click browse servers in the menu i dont know what u did but u didnt download the same version i think from kat.cr

**Comment by Guad** [comment]
Is there a GTACOOP.log in the scripts folder?

**Comment by dm104** [comment]
Yes inside it is writen:
<?xml version="1.0"?>
<PlayerSettings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <DisplayName>RLD!</DisplayName>
  <MaxStreamedNpcs>10</MaxStreamedNpcs>
  <MasterServerAddress>http://46.101.1.92/</MasterServerAddress>
  <ActivationKey>F9</ActivationKey>
</PlayerSettings>

**Comment by Guad** [comment]
Well post it then

**Comment by dm104** [comment]
Hire is the picture sory if i copy paste the code it doesnt show whole in my comment.
http://shrani.si/f/1s/eX/2PHwb9J2/brez-naslova.png

**Comment by Guad** [comment]
That's not GTACOOP.log, that's GTACOOPSettings.xml

**Comment by dm104** [comment]
Sory my fault i only have GTACOOPSettings and GTACoOp.dll when i search the whiole GTA5 instalation folder

**Comment by dm104** [comment]
So what should i do can you send me the files that are mising?

**Comment by Guad** [comment]
It's not missing. Try going to http://46.101.1.92/ and connect to any of the servers.

**Comment by dm104** [comment]
I wil try it in a sec.

**Comment by dm104** [comment]
After typing in the IP and presing connect i cant get back to the menu with F9 and i think nothing is connected because i cand find nobody inside noting realy happens.

**Comment by dm104** [comment]
Jup it says you have ben disconected .... after a while.

**Comment by dm104** [comment]
Tryed 10 difrend servers conection failed you have ben disconected no response from remote host so eather my version is difrend from yours or something firewall is not blocking anything....

**Comment by rakion99** [comment]
last resort buy the game and try again

**Comment by white-energy** [comment]
Don't trick him into thinking that buying the game will solve the issue.

**Comment by dm104** [comment]
My first game purchase was indiana jones an,the emporors tomb back in 2004 and my last was css series on steam sales for 5€ or so.

**Comment by dm104** [comment]
Any other ideas anyone i wil try it or should i wait for an script update?

**Comment by Guad** [comment]
Well it's something on your end that's blocking the servers. The master server is working fine since there is no GTACOOP.log.

**Comment by dm104** [comment]
Hmm i will try to play with my router setings.

**Comment by zain-masood** [comment]
All GTA 5 Cheats <a href="http://hackingevent.com/gta-5-cheats-codes-ps3-grand-theft-auto-v-secrets/">GTA 5 Cheats Ps3</a>

**Comment by Bluscream** [comment]
https://www.instant-gaming.com/en/186-buy-key-grand-theft-auto-v/ GTA V R* Version for under 30$
